dashboards-observability/public/components/application_analytics/components/app_table.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    EuiButton,
+    EuiContextMenuItem,
+    EuiContextMenuPanel,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiHorizontalRule,
+    EuiInMemoryTable,
+    EuiLink,
+    EuiPage,
+    EuiPageBody,
+    EuiPageContent,
+    EuiPageContentHeader,
+    EuiPageContentHeaderSection,
+    EuiPageHeader,
+    EuiPageHeaderSection,
+    EuiPopover,
+    EuiSpacer,
+    EuiTableFieldDataColumnType,
+    EuiText,
+    EuiTitle,
+} from '@elastic/eui';
+import CSS from 'csstype';
+import _ from 'lodash';
+import React, { ReactElement, useEffect, useState } from 'react';
+import { AppAnalyticsComponentDeps, ApplicationType } from '../home';
+
+const pageStyles: CSS.Properties = {
+  float: 'left',
+  width: '100%',
+  maxWidth: '1130px',
+};
```

Comment:
this appeared multiple times (also in notebooks), maybe define in constant files?

Reply:
Is it okay to change the pageStyle width to 1130 and panelStyle width to 1110?? This aligns with the rest of the plugins.
![Screen Shot 2021-12-13 at 11 37 19 AM](https://user-images.githubusercontent.com/92330893/145876965-6e7d5538-b9cc-4e90-a70e-246d5460ed38.png)

Reply:
sure if it doesn't look broken

Reply:
Didn't change the one on notebooks